packages/dds/merge-tree/src/test/client.localReference.spec.ts

```diff
@@ -695,6 +699,8 @@ describe("MergeTree.Client", () => {
 
 				segInfo.segment.localRefs.removeLocalRef(localRef);
 				assert(segInfo.segment.localRefs.empty);
+				// Cast is necessary because LocalReference is not exported, so we can't directly call link
```

Comment:
Nit: Can we cast to a specific type instead of `any`?

Reply:
I don't think we can, since the type we'd need to cast to would be LocalReference. I don't think we should export that just to avoid using `any` here, but I'm open to opinions.